cdap-securestore-ext-cloudkms/src/main/java/co/cask/cdap/securestore/gcp/cloudkms/CloudKMSClient.java

```diff
@@ -142,15 +146,15 @@ void createKeyRingIfNotExists() throws IOException {
     try {
       cloudKMS.projects().locations().keyRings()
         .create(parent, new KeyRing())
```

Comment:
Can we somewhere trace/debug log the keyring id for debugging if needed?

Also general question: what is the behavior if they keyring does not exist?

Reply:
Key ring is created if it does not exist. That behavior is not changed: https://github.com/cdapio/cdap/blob/develop/cdap-securestore-ext-cloudkms/src/main/java/co/cask/cdap/securestore/gcp/cloudkms/CloudKMSClient.java#L139

Reply:
Added debug log statement